docs/source/asr/speaker_recognition/models.rst

```diff
@@ -11,6 +11,20 @@ The Checkpoints page also contains benchmark results for the available speaker r
 
 .. _SpeakerNet_model:
```

Comment:
Is titanet belong to speakernet collection?

Reply:
This line is not necessary actually. Removed